src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/FlowCacheService.java

```diff
@@ -108,4 +110,8 @@ private void checkFlowLatency(String flowId, FlowState flowState) {
                     flowState.getMaxLatency(), flowState.getMaxLatencyTier2());
         }
     }
+
+    public boolean isExpired(Instant timestamp, Instant current, Duration expirationTime) {
```

Comment:
Do you really need `expirationTime` argument here? It is accessible as `flowRttStatsExpirationTime` class field.